windows/devices/uwb/include/windows/devices/uwb/UwbDeviceConnector.hxx

```diff
@@ -48,14 +50,31 @@ public:
      * @return false If listening for notifications could not be started.
      */
     bool
-    NotificationListenerStart(std::function<void(::uwb::protocol::fira::UwbNotificationData)> onNotification);
+    NotificationListenerStart();
 
     /**
      * @brief Stop listening for notifications.
      */
     void
     NotificationListenerStop();
 
+    /**

```

Comment:
I really like this interface, it's simple and accomplishes exactly what we need.